src/account/utils/signature.cairo

```diff
@@ -8,6 +8,9 @@ use starknet::eth_signature::Signature;
 use starknet::secp256_trait::{is_signature_entry_valid, recover_public_key};
 use starknet::secp256k1::Secp256k1Point;
 
+/// This function is intentionally not protected against signature maleability
+/// for efficiency reasons. It is not recommended to use it other than for
+/// validating account signatures over transaction hashes.
```

Comment:
```suggestion
/// This function assumes the `s` component of the signature to be positive 
/// for efficiency reasons. It is not recommended to use it other than for
/// validating account signatures over transaction hashes since otherwise
/// it's not protected against signature malleability.
/// See https://github.com/OpenZeppelin/cairo-contracts/issues/889.
```
it sounds we're intentionally vulnerable to malleability 😅
although somewhat true, I think this is a better angle

Reply:
That quite an improvement :)